react/src/components/Table/Table.tsx

```diff
@@ -364,6 +317,18 @@ const Table: React.FC<TableProps> = ({ variantData }) => {
                         width: 79,
                     },
                     { accessor: 'transcript', id: 'transcript', Header: 'transcript', width: 150 },
+                    {
+                        accessor: 'homozygousCount',
+                        id: 'homozygousCount',
+                        Header: 'homozygous count',
```

Comment:
Capitalize: Homozygous Count

Reply:
Just to confirm, are we sure about this? All the other headers are not capitalized.

Reply:
Yup! I think the other ones are not capitalized (like `gnomadHom`) because it's the standard way the gnomAD database writes it